tests/TestMerge.cpp

```diff
@@ -564,6 +565,29 @@ void TestMerge::testResolveGroupConflictOlder()
     delete dbSource;
 }
 
+/**
+  * If identical databases are merged, the modified signal
+  * should not be emitted. If different databases are merged,
+  * the modified signal should be emitted.
+  */
+void TestMerge::testModifiedEmission()
+{
+    Database* dbDestination = createTestDatabase();
+    QSignalSpy spy(dbDestination, SIGNAL(modified()));
```

Comment:
@mbilloo We should check that the modification date of the database has not changed. That would also remove the need to spy on `modified`.

Reply:
This particular test was created to verify that the modified() signal is not emitted when merging identical databases and emitted when merging different databases, to ensure that the changes made to the source are correct for all possible scenarios. 

Any other conditions for which the databases should not be merged should be as separate issues and separate test cases.